utils/avrdude/Makefile

```diff
@@ -29,14 +29,18 @@ define Package/avrdude
   SUBMENU:=Microcontroller programming
   TITLE:=AVR Downloader/UploaDEr
   URL:=https://github.com/avrdudes/avrdude
-  DEPENDS:=+libncurses +libreadline +libusb-1.0 +libftdi1 +libelf
+  DEPENDS:=+libncurses +libreadline +libusb-1.0 +libftdi1 +libelf +libgpiod
```

Comment:
Is `libftd1` required if `-DHAVE_LIBFTDI=OFF`?

Reply:
yes, libftdi1 and libftdi are two different libraries